### PR TITLE
space-age: Update description.md

### PR DESCRIPTION
**Pull request by fedavies**
Clarification of the English in this problem specification.

### CODE REVIEW

File: exercises/space-age/description.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Given an age in seconds, calculate how old someone would be on:
+ Given an age in seconds, calculate that age in terms of years of the different planets. Here is a list of the length of one 'year' on each planet:
```
Comment (rpottsoh) [inline exercises/space-age/description.md]: Just a nit pick, but it appears like there is a leading space that isn't needed.  Also, `"that"` sounds strange to me.  How about `a persons`?  It isn't clear what "that" is referring to.  Is it meant to be the age of the planet, or in the context of the original description, a persons age on a given planet?

Comment (wolf99) [reply exercises/space-age/description.md]: Both sound a little odd to me!
Maybe a more concise version would be
`Given a person's age in seconds, calculate their age in terms of years of the different planets. ...`

Comment (fedavies) [reply exercises/space-age/description.md]: I agree that what you have written is clearer

**Comment by rpottsoh** [comment]
@fedavies just a friendly nudge.  Do you need assistance in making the suggested changes?

**Comment by fedavies** [comment]
I might need a hand, thanks. I think that I have done it now but I'm not sure - I'm new to all this!

File: exercises/space-age/description.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Given an age in seconds, calculate how old someone would be on:
+Given a person's age in seconds, calculate their age in terms of years of the different planets.
```
Comment (Insti) [inline exercises/space-age/description.md]: > a person's 

That it's a person is largely irrelevant.

> years of the different planets

This does not make grammatical sense to me.


Disclaimer: I don't think there is anything wrong with the original wording.

Comment (wolf99) [reply exercises/space-age/description.md]: > That it's a person is largely irrelevant.

True, but supplying an object for the noun makes it easier for students to understand by adding a little more context. Of course _we_ know what it means as-is, but people that don't already know may struggle - as evidenced by the fact that the issue was raised.

>>years of the different planets
>
>This does not make grammatical sense to me.

Maybe this should be "years _on_ the different planets"? Or maybe even "according to the length of a year on a different planet"

**Comment by kotp** [comment]
Thank you @fedavies even if this is not merged in, it shows good effort and we appreciate it.

**Pull request closed without merging**